src/users/users.controller.ts

```diff
@@ -60,7 +60,7 @@ export class UsersController {
   @ApiNotFoundResponse({
     description: 'User not found',
   })
-  @Get(':id')
+  @Get('user/:id')
```

Comment:
Hmm? Why add `user` again, and can't we use this endpoint to return profile too? There's no different after all.

Reply:
Oh don't mind about merging those endpoint, but I think we should create a response dto that will be used for both endpoint because both have the same purpose so maybe always same response. What do you think?

Reply:
> Hmm? Why add `user` , and can't we use this endpoint to return profile too? There's no different after all.

@Get(':id') make you other get endpoint like @Get('profile') will call @Get(':id') that id = 'profile'

Reply:
> Oh don't mind about merging those endpoint, but I think we should create a response dto that will be used for both endpoint because both have the same purpose so maybe always same response. What do you think?

Ok